tests/foreman/rhai/test_rhai.py

```diff
@@ -130,6 +131,9 @@ def test_unregister_system_from_rhai(self):
                 session.nav.click(
                     locators['insights.unregister_button']
                 )
+                self.browser.refresh()
+                time.sleep(30)
```

Comment:
As @elyezer mentioned, hard sleep is really bad idea and especially having static amount of seconds. Is it possible verify your conditions dynamically in some way with smaller delay?
Something like:
```
for n tries
    check whether condition is true
    if not  - sleep for let's say 5 seconds
```